buildSrc/src/main/kotlin/ktlint.gradle.kts

```diff
@@ -16,42 +18,29 @@
  * SPDX-License-Identifier: Apache-2.0
  *
  */
+plugins {
+    id("org.jlleitschuh.gradle.ktlint")
+}
 
-val ktlint by configurations.creating
+// https://github.com/jlleitschuh/ktlint-gradle#configuration
+ktlint {
+    version.set("1.0.1")
```

Comment:
This is the [default value](https://github.com/JLLeitschuh/ktlint-gradle/blob/main/plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt#L39). Line could be removed so that the ktLint version is updated with the corresponding gradle plugin.

Reply:
No strong opinion here. We can keep it like this for now.